indico/modules/categories/blueprint.py

```diff
@@ -31,8 +32,13 @@
 _bp.add_url_rule('/manage/icon', 'upload_icon', RHSettingsIconUpload, methods=('POST', 'DELETE'))
 _bp.add_url_rule('/manage/protection', 'manage_protection', RHManageCategoryProtection, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/settings', 'manage_settings', RHManageCategorySettings, methods=('POST', 'GET'))
+_bp.add_url_rule('/manage/move', 'move-contents', RHCategoryMoveContents)
```

Comment:
Not sure if it makes sense to have the same RH for moving events and subcategories. The logic is different for sure.